src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -282,7 +286,8 @@ SHAMapStoreImp::run()
     while (1)
     {
         healthy_ = true;
-        validatedLedger_.reset();
+        validatedLedger.reset();
```

Comment:
It _does_ seem like a lot of effort to go through to avoid simply declaring `Ledger::pointer validatedLedger;`